### ISSUE TITLE
toolsblocklight.vmat no longer blocks light in Hammer maps.

### ISSUE THREAD

**Issue by Doodl0** [Facepunch/sbox-issues#6298]
### Describe the bug

Up until recently, I have been using toolsblocklight.vmat to block sunlight from indoor areas in my map. However, this has stopped working, even after a full recompile of the map.

### To Reproduce

Make a room with toolsblocklight around it and a light source outside it, such as an environment light, compile it, and see if toolsblocklight kept the room dark.

### Expected behavior

toolsblocklight.vmat should block light in maps.

**Comment by QuackCola** [comment]
Ideally we should be doing this instead https://github.com/Facepunch/sbox-issues/issues/3495

**Comment by MrBrax** [comment]
blocklight is useful for other scenarios but i agree that backface blocking should be a thing

**Comment by Doodl0** [comment]
Just to add to this, I have tried toolssolidblocklight.vmat and it also does not block light. I also made sure that the faces of the mesh with toolsblocklight are not inverted and face the light source, and it still does not block light

**Comment by AfteraOfficiel** [comment]
I confirm that it really bugs, sometimes there is no more shadow on complete area... while a few meters further the shadow is there

![Image](https://github.com/user-attachments/assets/0e35305e-df57-4cb1-a9b0-307f53c372e0)
![Image](https://github.com/user-attachments/assets/9a09daa1-e2c5-4d4f-899f-b170595b91a1)

**Comment by Doodl0** [comment]
> I confirm that it really bugs, sometimes there is no more shadow on complete area... while a few meters further the shadow is there
> 
Yeah, it's a shame this still hasn't been fixed after 3 months. Take this with a grain of salt, but I think it's due to the real time lighting being part of the scene, not the map, so the scene just renders it through the map.

**Comment by Nolankicks** [comment]
This would all be fixed if vrad renders backlights.

**Comment by Doodl0** [comment]
Yes, but toolsblocklight is still useful for other things, not just backface blocking
